src/App.tsx

```diff
@@ -11,11 +11,19 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => {
+      return {
+        count: state.count + 1,
+      };
+    });
```

Comment:
`return` is redundant here, you can return the object without it
```suggestion
    this.setState((state) => ({
      count: state.count + 1,
    }));
```